pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -146,9 +152,21 @@ func testSyncNamespaceThatIsTerminating(t *testing.T, versions *unversioned.APIV
 
 	for scenario, testInput := range scenarios {
 		mockClient := fake.NewSimpleClientset(testInput.testNamespace)
+		if containsVersion(versions, "autoscaling/v1") {
+			resources := []unversioned.APIResource{}
+			for _, resource := range []string{"horizontalpodautoscalers"} {
+				resources = append(resources, unversioned.APIResource{Name: resource})
+			}
+			mockClient.Resources = map[string]*unversioned.APIResourceList{
+				"autoscaling/v1": {
+					GroupVersion: "autoscaling/v1",
+					APIResources: resources,
+				},
+			}
+		}
 		if containsVersion(versions, "extensions/v1beta1") {
 			resources := []unversioned.APIResource{}
-			for _, resource := range []string{"daemonsets", "deployments", "jobs", "horizontalpodautoscalers", "ingresses"} {
```

Comment:
I'd like to remove support for HPA extensions in 1.2 to not encourage users to use extensions version instead of autoscaling. That's why I removed support for it in extensions client. As I described I'd like to still support for `extensions/v1beta1` on api server level but I'd like to not use this version internally anymore.